src/index.js

```diff
@@ -25,6 +26,7 @@ const routing = (
       <Route path="/forgotpasswordpagesend" component={ForgotPasswordPageSend} />
       <Route path="/forgotpasswordpageinput" component={ForgotPasswordPageInput} />
       <Route path="/forgotpasswordpagechange" component={ForgotPasswordPageChange} />
+      <Route path="/websitepage" component={WebsitePage}/>
```

Comment:
can we change this to `<Route path="/" component={WebsitePage}/>`?

Reply:
Yeah I was just unsure of what "HomePage" was, the current path="/" routes to that.